AudioSensei/WebHelper.cs

```diff
@@ -2,11 +2,13 @@
 using System.Net;
 using System.Net.Http;
 using System.Runtime.InteropServices;
+using DynamicData.Annotations;
```

Comment:
Are you sure?

Reply:
There are 3 different [NotNull] and [CanBeNull] annotations there, in three different namespaces.
Usings were added by Rider, after I explicitly clicked which one, might've made a small mistake
```suggestion
using JetBrains.Annotations;
```